src/ui/server/helpers/mappings/map-application-to-form-fields/index.ts

```diff
@@ -74,6 +76,13 @@ const mapApplicationToFormFields = (application?: Application): Application | ob
       };
     }
 
+    if (application?.policy?.[POLICY_CURRENCY_CODE]) {
```

Comment:
do we need this line as won't it always be set to a default value of GBP so will always exist?